vizro-core/changelog.d/20230911_180805_huong_li_nguyen_enable_targeting_ids.md

```diff
@@ -0,0 +1,42 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category.
+
+-->
+<!--
+### Changed
+
+- Provide ID to outer div to enable CSS customization of component and its sub-components (#29)
```

Comment:
You need to uncomment the block so that it actually shows up :) (see explanation at the top of the file), could we also provide the hyperlink to the PR? For added snazziness :)

Reply:
Can do - I'll change the PR template then as well.